scripts/docker/CC-Devel/bin/install_gtest.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+pushd /usr/src/gtest
+cmake .
+make
+cp libgtest.a libgtest_main.a /usr/lib
```

Comment:
If we use custom build directories such as `/opt/odb`, it would be much more in-line with this approach if building gtest didn't "litter" the root.

(I think I have updated the dependency guide in my recent patch to cover this.)